origami.json

```
@@ -1,6 +1,6 @@
 {
-	"description": "A footer for non-master brand FT products",
-	"keywords": "footer, note, legal, copyright, logo, contact",
+	"description": "This is a legally approved footer component for internal products, tooling and customer products that are not FT.com",
```

Makes sense to us, but this might be better to avoid confusion as internal tools are sometimes on an ft.com subdomain?
```suggestion
	"description": "This is a legally approved footer component for internal products, tooling and customer products",
```